src/js/components/Chart.vue

```diff
@@ -228,10 +237,12 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@use '../../sass/app';
```

Comment:
I'm a bit confused by this change. Now that this `@use` is nested inside a `scoped` block, these styles are no longer global -- they'll be scoped to this specific `Chart` component.

Reply:
@mirisuzanne this might be a dumb question, but do we need the app styles at all? I believe the user agents all set overflow to hidden (`svg:not(:root)`)  and the only thing that I can tell that needs the border-box is the Dev.vue form, and that already has it in the css at the bottom.

At the very least, I don't think we need to use app here.